src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -279,6 +279,8 @@ Convolution::Convolution(const std::shared_ptr<ngraph::Node>& op, const dnnl::en
         paddingR = groupConvolutionOp->get_pads_end();
         autoPadding = one_of(groupConvolutionOp->get_auto_pad(), ov::op::PadType::SAME_UPPER, ov::op::PadType::SAME_LOWER);
     }
+
+    shouldTryBrgconvAVX512 = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx);
```

Comment:
I think `AVX512` is redundant in the name given the same flag is used both for AMX and AVX512 cases.

Reply:
The suffix removed.